installed-tests/minijasmine.cpp

```diff
@@ -33,6 +33,8 @@
 #include "gjs/gjs.h"
 #include "gjs/mem.h"
 
+G_GNUC_NORETURN void bail_out(const char *msg);
+
 G_GNUC_NORETURN
 void
```

Comment:
Instead of adding the declaration, I think the better solution is to make this function `static void` since it's not used in any other files.